.pre-commit-config.yaml

```diff
@@ -12,14 +12,14 @@ repos:
     -   id: rust-style
         name: Rust style
         description: Run cargo fmt on files included in the commit. rustfmt should be installed before-hand.
-        entry: cargo fmt --all -- --check
+        entry: cargo +nightly-2021-11-26 fmt --all -- --check
```

Comment:
is the data really requiererd? If so, thats going to be in PITA

Reply:
Something else is also broken:

```
+ nix-shell --run 'cargo fmt --all -- --check' ci.nix

error: attribute '2021-11-26' missing, at /var/lib/jenkins/workspace/Mayastor_trying/nix/lib/rust.nix:7:13

(use '--show-trace' to show detailed location information)

script returned exit code 1
```

Reply:
Fixed.